src/Contract.php

```diff
@@ -22,8 +29,35 @@ class Contract extends Model
      */
     public function __construct(array $attributes = [])
     {
-        $this->fillables('file', 'status', 'signed_at', 'options');
-        $this->columns('file', 'status', 'signed_at', 'options');
+        $this->fillables('file', 'status', 'signed_at', 'options', 'template', 'external_id', 'user_id');
+        $this->columns('file', 'status', 'signed_at', 'options', 'external_id', 'user_id');
         parent::__construct($attributes);
     }
+
+    public function toPdf(HtmlConverterInterface $htmlConverter)
+    {
+        if (!$this->template) {
```

Comment:
add a config to mvc-contracts: pdf.enabled: true/false (filled via .env)
check here in the if statement for the enabling of the service